storagenode/retain/retain.go

```diff
@@ -205,13 +310,13 @@ func (s *Service) Status() Status {
 // written to the filesystem _after_ the CreationTime. We make the assumption already that storage
 // nodes and satellites and uplinks have system clocks that are very roughly in sync (that is, they
 // are out of sync with each other by less than an hour of real time, or whatever is configured as
-// RetainTimeBuffer). So if an uplink is not lying about CreationTime and it uploads a piece that
+// MaxTimeSkew). So if an uplink is not lying about CreationTime and it uploads a piece that
```

Comment:
Would it be better to adjust by `MaxTimeSkew` inside `retainPieces` rather than inside `Queue`? It would probably make more sense in the context of this comment.